### PR TITLE
Enable debugging info only if logging option is enabled.

### PR DESCRIPTION
**Pull request by yurlovm**
Consider to move console.log under configuration flag, please.

### CODE REVIEW

**Comment by HNeukermans** [comment]
Nice PR.  This is a  certainly an improvement.
However if you could move the logging to a separate function, I will be glad to merge it in. 

`
private function log(args){
   if(this._Connection.logging == false) return;
   console.log(args)
}
`

**Comment by yurlovm** [comment]
Can I fix some linting stuff too?

**Comment by HNeukermans** [comment]
@yurlovm,

Fix linting issues? Offcourse. that would be even better.

Thx in advance..

**Comment by yurlovm** [comment]
Hello, check this please. Yes, vscode with tslint lights some suggestions.

File: src/services/signalr.ts [hunk 1]
```diff
@@ -20,24 +20,23 @@ export class SignalR {
...
                 if (configuration.logging) {
+                    console.log(`Connecting with...`);
+                    console.log(`configuration:[url: '${configuration.url}'] ...`);
+                    console.log(`configuration:[hubName: '${configuration.hubName}'] ...`);
                     console.log(`configuration:[qs: '${serialized}'] ...`);
                 }
-            } catch (err) { }
+
+            } catch (err) {
+                console.log(err);
```
Comment (HNeukermans) [inline src/services/signalr.ts]: We can silently ignore serialization errors for logging. Could you remove this line ?

**Comment by yurlovm** [comment]
Removed.
Btw I tried to use your ng2-signalr in reactive way, if you interested check [this ](https://github.com/yurlovm/asp.net_4.6_angular_webpack/blob/master/WebApiAngular/src/app/core/services/signalr.service.ts) 
I'm not expert in RFP, but it works :)

**Comment by HNeukermans** [comment]
Nice work you got there : @yurlovm :  asp.net_4.6_angular_webpack.
.NET - Angular 2 :+1:

**Comment by yurlovm** [comment]
thank you :)

**Comment by HNeukermans** [comment]
Nice RFP skills you got there. I learned something...

**Comment by yurlovm** [comment]
Thanks, actually I asked some experts to look at it too. If they advise to change something I can ping you after code changes )

**Pull request merged**